packages/next/build/entries.ts

```diff
@@ -493,6 +498,7 @@ export async function createEntrypoints(params: CreateEntrypointsParams) {
             middleware: staticInfo?.middleware,
             pagesType,
             appDirLoader,
+            userConfig: staticInfo?.userConfig,
```

Comment:
For consistency I think we may want to have this be `config` and only add expected config fields which are defined in our type definition.

Reply:
I kept the entire config so we will be able to serialize what we want in `MiddlewarePlugin`, but changed it to serialize only `regions`. Does that make sense? <picture data-single-emoji=":pray_parrot:" title=":pray_parrot:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..jXB0Kt84_yOPEIKL.SaDhaX5tiJJb7ZLjSfwra65tDcbzX5LfYghcnk5ziZ6Nbe5BX4ezQ9BQOkM0TTOFoFPoCU2FC5kL4vRtAIr9lexizzypVeLYBf2o-Kt7ZeZafUKz5WGtiBoc9qo.eAiw9d7XVjm1udonO_PbZw" alt=":pray_parrot:" align="absmiddle"></picture>